core/domain/role_services.py

```diff
@@ -49,18 +51,21 @@
 ACTION_CREATE_NEW_TOPIC = 'CREATE_NEW_TOPIC'
 ACTION_MANAGE_QUESTION_SKILL_STATUS = 'MANAGE_QUESTION_SKILL_STATUS'
 ACTION_DELETE_ANY_ACTIVITY = 'DELETE_ANY_ACTIVITY'
+ACTION_DELETE_ANY_BLOG_POST = 'DELETE_ANY_BLOG_POST'
 ACTION_DELETE_ANY_PUBLIC_ACTIVITY = 'DELETE_ANY_PUBLIC_ACTIVITY'
 ACTION_DELETE_ANY_QUESTION = 'DELETE_ANY_QUESTION'
 ACTION_DELETE_ANY_SKILL = 'DELETE_ANY_SKILL'
+ACTION_DELETE_OWNED_BLOG_POST = 'DELETE_BLOG_POST'
```

Comment:
I'm confused about this. Where do you use it?

Reply:
Removed.